packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -244,6 +245,11 @@ export interface S3OriginConfig {
    * The optional ID of the origin identity cloudfront will use when calling your s3 bucket.
    */
   readonly originAccessIdentityId?: string;
```

Comment:
Should this be dropped and instead be a breaking change?

Reply:
yes please. It's an experimental module, so you can break.

Reply:
@eladb it was not included as a breaking change in the release notes and just broke every website I am running ...

Reply:
Sorry! Apologies for the bad experience. I'll update the release notes in the changelog and on GitHub.

Reply:
Updated. Thanks for letting us know @hoegertn and sorry again for the hassle.